Cargo.toml

```diff
@@ -43,7 +46,7 @@ quickcheck = "0.9"
 criterion = "0.3"
 
 [features]
-default = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld", "jpeg_rayon"]
+default = ["avif", "gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld", "jpeg_rayon"]
```

Comment:
The feature can't be default, as it conflicts with the MSRV policy. I'd be happy for a note though and to add it in the next major release.

Reply:
Thanks, this is fixed in https://github.com/image-rs/image/pull/1398/commits/84c72fee412ca6f68e9e88d6199077ad72166a7e

Reply:
> The feature can't be default, as it conflicts with the MSRV policy. I'd be happy for a note though and to add it in the next major release.

Making `avif` a default feature will require a modern version of `nasm` to be present aside from a rust toolchain, since AVIF encoding depends on `ravif`, which requires a recent version of `nasm` to be present (see https://github.com/image-rs/image/issues/1404#issuecomment-769117536), at least by default.